package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "cabal-phantom-frontend",
+  "version": "1.0.0",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack serve --config ./webpack.config.js --mode development",
```

Comment:
you don't need to specify the config file since you named it right
so this would just do it "webpack serve --mode development"